unlock-app/src/services/web3Service.js

```diff
@@ -110,18 +108,9 @@ export default class Web3Service extends EventEmitter {
       },
     }
 
-    if (unlockContractAddress) {
-      this.unlockContractAddress = Web3Utils.toChecksumAddress(
-        unlockContractAddress
-      )
-    } else if (UnlockContract.networks[requiredNetworkId]) {
-      // If we do not have an address from config let's use the artifact files
-      this.unlockContractAddress = Web3Utils.toChecksumAddress(
-        UnlockContract.networks[requiredNetworkId].address
-      )
-    } else {
-      return this.emit('error', new Error(NON_DEPLOYED_CONTRACT))
-    }
+    this.unlockContractAddress = Web3Utils.toChecksumAddress(
```

Comment:
Maybe we can move this to config.js since it is now static?

Reply:
That's an option. I think we can even put the checksumed address inside the config so that we do not even have to checksum it.